lib/exec.ml

```diff
@@ -126,6 +126,46 @@ let git_resolve ~remote ~ref =
       Rresult.R.error_msgf "Multiple refs share the name %a on the remote %s" Git.Ref.pp ref remote
   | Error (`Msg _) as err -> err
 
+let git_resolve_local ~repo ~ref =
+  run_and_log_l Cmd.(v "git" % "-C" % p repo % "show-ref" % ref) >>= fun output ->
```

Comment:
Is this command is expected to fail ?
If yes, errors should not be logged as ERROR (`~ignore_error:true`).
I'm getting 2 screens of:

```txt
duniverse: [ERROR] git -C
                     /home/jules/.cache/duniverse/https---github.com-janestreet-sexplib0.git
                     show-ref branch-v0.12.0 failed. Output was:
```